starlette/websockets.py

```diff
@@ -13,26 +16,26 @@ class WebSocketState(enum.Enum):
 
 
 class WebSocketDisconnect(Exception):
-    def __init__(self, code=1000):
+    def __init__(self, code: int = 1000) -> None:
         self.code = code
 
 
 class WebSocket(Mapping):
-    def __init__(self, scope, receive=None, send=None):
+    def __init__(self, scope: Scope, receive: Receive, send: Send) -> None:
```

Comment:
Is it possible to leave `send` and `receive` as optionals for now, or does that introduce errors?

Reply:
If we allow optional like before we have ` error: "None" not callable`. Anyway, does it make sense that `receive` and `send` can be optional? If for some reason any of them is `None` the server will crash.